client/src/services/notificationCenter.ts

```diff
@@ -92,7 +107,7 @@ export class NotificationCenter {
     return modal.onWillDismiss();
   }
 
-  async showSnackbar({
+  async showToast({
```

Comment:
I'm guessing the `{}` are there to be able to set `addToList` or `trace` to true, like `showToast({notification, trace: true})` instead of doing `showToast(notification, false, true)` because typescript does not have real named arguments so you can't just do `showToast(notification, trace=true)`. Is that it?

Not being able to just do `showToast(notification)` kinda sucks, wouldn't it be better to have
`showToast(notification, {addToList = false, trace = false}`? I'm guessing this would work (not sure but I don't see why not) and would allow us to just do `showToast(notification)`, or `showToast(notification, {trace: true})` in case we want to use the other parameters.